src/sentry/db/mixin.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
+
+import logging
+import six
+
+from time import time
+
+from sentry.models import OrganizationOption
+
+logger = logging.getLogger('sentry.deletions')
+
+
+def delete_pending_deletion_option(instance, **kwargs):
+    if hasattr(instance, 'delete_pending_deletion_option'):
+        instance.delete_pending_deletion_option()
+
+
+class PendingDeletionMixin(object):
+    _rename_fields_on_pending_delete = []
```

Comment:
This isn't an problem today, but I'd shy away from using `[]` here since it's mutable and at the class level. This means a mutation can be shared between all instances or all classes that use this mixin. It'd be safer to either use an empty tuple or just `None`.

Reply:
@mattrobenolt would a `frozenset` make more sense?